Python/bytecodes.c

```diff
@@ -981,7 +988,7 @@ dummy_func(
                 tstate->exc_info = &gen->gi_exc_state;
                 SKIP_OVER(INLINE_CACHE_ENTRIES_SEND);
                 assert(1 + INLINE_CACHE_ENTRIES_SEND == next_instr - frame->instr_ptr);
-                frame->next_instr_offset = 1 + INLINE_CACHE_ENTRIES_SEND + oparg;
+                frame->return_offset = 1 + INLINE_CACHE_ENTRIES_SEND + oparg;
```

Comment:
So this line causes a warning about integer size in MSVC in the generated code (and also the similar line in `SEND_GEN` below). I'm guessing it needs a cast?